docs/documentation/pom.xml

```diff
@@ -145,7 +144,7 @@
                 <plugin>
                     <groupId>org.asciidoctor</groupId>
                     <artifactId>asciidoctor-maven-plugin</artifactId>
-                    <version>${version.plugin.asciidoctor}</version>
+                    <version>${asciidoctor.plugin.version}</version>
```

Comment:
As the version is pinned via `pluginManagement` in the parent pom, the version line can (and should) be removed.

```suggestion
```